assets/js/util/refresh-authentication.js

```diff
@@ -31,7 +31,7 @@ export const refreshAuthentication = async () => {
 
 		// We should really be using state management. This is terrible.
 		global.googlesitekit.setup = global.googlesitekit.setup || {};
-		global.googlesitekit.setup.isAuthenticated = response.isAuthenticated;
+		global.googlesitekit.setup.authenticated = response.authenticated;
```

Comment:
I think this is the only line in JS legacy code that should be updated based on the renaming. `global.googlesitekit.setup.isAuthenticated = response.authenticated;` would be sufficient to keep the old name for the legacy JS data (which will be phased out in the long term) while supporting the new name in the API route.

All updates to the JS components above wouldn't be necessary.